src/injections/withStatusHandling/README.md

```diff
@@ -0,0 +1,27 @@
+## withStatusHandling
+
+The `withStatusHandling` injector allows to describe some behavior based on the response status code.
+`withStatusHandling` recieves an object whose property names are the different http status codes (404, 200, etc) and that define the logic to execute for the desired status code.
```

Comment:
Actually `200` is not being handled here by default, since it falls under `response.ok` category. This by default handles those that caused a failure in the response (4xx, 5xx)